doc/tutorial/Authentication.lhs

```diff
@@ -246,7 +246,7 @@ your feedback!
 ### What is Generalized Authentication?
 
 **TL;DR**: you throw a tagged `AuthProtect` combinator in front of the endpoints
-you want protected and then supply a function `Request -> Handler user`
```

Comment:
It seems like `user` is *less* misleading, no? `req` to me means "request"

Reply:
Ah right, my bad, as it's a `Handler` of some 'application-specific datatype'. Would it be better to say something like `Request -> Handler a` or use lowercase `account` to keep in line with the subsequent example?

Reply:
`a` sounds best to me, but maybe all of this indicates that we could use another sentence of explanation (something like "where `a` is a type of your choice representing the data returned by successful authentication - e.g., a `User` or, in our example below, `Account`")

Reply:
I was thinking the same (for the latter sentence) otherwise people would have to read on to connect the two. I think your copy is quite apropos, so I'll tack that on.